CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 7.0.0 (2022-04-23)
+
+This version contains **breaking** changes, as bugsnag-unity has been updated to use a more advanced Exception interceptor and existing grouping between errors in the Bugsnag dashboard may be broken.
+
+Please see the [upgrade guide](./UPGRADING.md) for details of all the changes and instructions on how to upgrade.
+
+In addition to the changes mentioned in the upgrade guide, the bundled Bugsnag Android Notifier has been updated. See below for details.
+
+* Update bugsnag-android to v5.22.3
```

Comment:
Maybe a good time to adopt a more brief syntax for this?

https://github.com/bugsnag/bugsnag-unreal/blob/next/CHANGELOG.md?plain=1#L13-L14

Reply:
Done 👍